src/Semantic/Api/Symbols.hs

```diff
@@ -26,11 +28,12 @@ import           Serializing.Format
 import           Source.Loc
 import           Tags.Taggable
 import           Tags.Tagging
+import qualified Tags.Tagging.Precise as Precise
```

Comment:
This will eventually be just `Tags.Tagging`—i.e. we’ll make à la carte tagging use `semantic-tags` too, or get rid of à la carte syntax altogether, whichever comes sooner.